userspace/falco/app/options.cpp

```diff
@@ -216,7 +216,7 @@ void options::define(cxxopts::Options& opts)
 		("nodriver",                      "Do not use a driver to instrument the kernel. If a loaded plugin has event sourcing capability and can produce system events, it will be used to for event collection.", cxxopts::value(nodriver)->default_value("false"))
 		("o,option",                      "Set the value of option <opt> to <val>. Overrides values in configuration file. <opt> can be identified using its location in configuration file using dot notation. Elements which are entries of lists can be accessed via square brackets [].\n    E.g. base.id = val\n         base.subvalue.subvalue2 = val\n         base.list[1]=val", cxxopts::value(cmdline_config_options), "<opt>=<val>")
 		("plugin-info",                   "Print info for a single plugin and exit.\nThis includes all descriptivo info like name and author, along with the\nschema format for the init configuration and a list of suggested open parameters.\n<plugin_name> can be the name of the plugin or its configured library_path.", cxxopts::value(print_plugin_info), "<plugin_name>")
-		("p,print",                       "Add additional information to each falco notification's output.\nWith -pc or -pcontainer will use a container-friendly format.\nWith -pk or -pkubernetes will use a kubernetes-friendly format.\nAdditionally, specifying -pc/-pk will change the interpretation of %container.info in rule output fields.", cxxopts::value(print_additional), "<output_format>")
+		("p,print",                       "Print (or replace) additional information in rule's output.\n - Use -c or --container to append container details.\n - Use -k or --kubernetes to add both container and Kubernetes details.\n - If using gVisor, choose -cg or -kg variants\n   (or --container-gvisor and --kubernetes-gvisor, respectively).\nIf a rule's output contains `%container.info`, it will be replaced with the corresponding details. Otherwise, these details will be directly appended to the rule's output.\nAlternatively, use -p \"...\" for a custom format. In this case, the given content will be appended to the \nrule's output without any replacement.", cxxopts::value(print_additional), "<output_format>")
```

Comment:
Sorry for always getting back to the `-k` flag confusion in general, here it should be used like `-pk` or `-p --kubernetes`, could we instead write `-pk` and `-pc` etc? And I wouldn't see any harm in restating it's different from the `-k` Kubernetes metadata fetching and maybe even restating once more here that in the kubernetes case it adds namespace and pod name and those fields are extracted from the container runtime ...

Reply:
Maybe even saying Falco does not behave like bash utils flag combination style ... in this context.

Reply:
Sorry, my bad. I meant `-pk`. I will fix it shortly.

Reply:
@incertum 
I fixed all the missing `p`. I haven't changed the whole message again since it's already quite long.
However, it's a WIP, so any suggestion is welcome.

PS
Thank you for catching this!

Reply:
This works ty! We can probably move this out of WIP now?

Reply:
Awesome, ty!